cypress/integration/lists_test/single_list_test.js

```diff
@@ -0,0 +1,71 @@
+describe("Single Lists Test", function() {
+    beforeEach(function() {
+      cy.visit("/");
+      cy.createGeneList("SODB, GBP, GST, CDPK1");
+    });
+
+    it("can copy a list", function(){
+        cy.url().should("include","/lists");
+        cy.get('.lists-item').should('have.length',1);
```

Comment:
Any reason why these lines can't be in the beforeEach, similar to multiple_lists_test?

Reply:
Nope, just added them to beforeEach!